tessellation/src/path_stroke.rs

```diff
@@ -437,21 +545,27 @@ fn get_angle_info(previous: Point, current: Point, next: Point) -> (Vec2, Vec2,
             }
         }
     };
-    return (inter - current, current - inter, None);
+    return inter - current;
+}
+
+/// Computes the max angle of a radius segment for a given tolerance
+pub fn compute_max_radius_segment_angle(radius: f64, tolerance: f64) -> f64 {
```

Comment:
Changed in https://github.com/nical/lyon/pull/93/commits/06e7a2b8915b7f6210b9ff36adbd4f2d3450e2bc